recipes/ubermagtable/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set version = "0.1.2" %}
+{% set name = "ubermagtable" %}
+{% set sha256 = "62c8920d08b1ac15da371c2df11293dcbefdd867fbf299f6fa9dd67fb353f110" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install --no-deps .
```

This sometimes makes debugging easier:
```suggestion
  script: pip install --no-deps . -vvv
```